phpstan.neon.dist

```diff
@@ -12,6 +12,10 @@ parameters:
 	ignoreErrors:
 		- '#Call to an undefined method CodeIgniter\\Database\\ConnectionInterface::[A-Za-z].+\(\)#'
 		- '#Cannot access property [\$a-z_]+ on (array|object)#'
+		- 
+			message: '#Call to deprecated function random_string\(\):#'
+			count: 5
+			path: *
```

Comment:
I think it is better to set path for all existing files, because the type 'basic', 'md5', and 'sha1' are really deprecated.
Setting `*` suppresses the error even when someone uses these type accidentally.

Reply:
Thanks, good point, done.